azure-pipelines.yml

```diff
@@ -12,9 +12,28 @@ steps:
 - script: npm run build
   displayName: npm run build
 
+# test using node 8.x
 - script: npm run units
   displayName: Run unit tests
 
+# test using node 10.x
+- task: NodeTool@0
+  inputs:
+    versionSpec: "10.x"
+  displayName: Install node 10
+
+- script: npm run units
+  displayName: npm run units
+
+# test using node 14.x
+- task: NodeTool@0
+  inputs:
+    versionSpec: "14.x"
+  displayName: Install node 14
+
+- script: npm run units
+  displayName: npm run units
```

Comment:
Maybe we could use a [strategy](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=example%2Cparameter-schema#strategies) for these steps? Since the only difference is the `Node` version.

Reply:
Yes, seems like a great idea, also will add matrix to the typed-rest-client tests as well